app/src/main/java/com/framgia/englishconversation/screen/dialog/OptionPostFragment.java

```diff
@@ -0,0 +1,66 @@
+package com.framgia.englishconversation.screen.dialog;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.BottomSheetBehavior;
+import android.support.design.widget.BottomSheetDialog;
```

Comment:
ERROR: Unused import: android.support.design.widget.BottomSheetDialog. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports